handlers/deploy_test.go

```diff
@@ -32,6 +32,25 @@ func Test_buildAnnotations_Empty_In_CreateRequest(t *testing.T) {
 	}
 }
 
+func Test_buildAnnotations_Premetheus_NotOverridden(t *testing.T) {
+	request := types.FunctionDeployment{Annotations: &map[string]string{"prometheus.io.scrape": "true"}}
+
+	annotations := buildAnnotations(request)
+
+	if len(annotations) != 1 {
+		t.Errorf("want: %d annotations got: %d", 1, len(annotations))
```

Comment:
You could probably add a return there, or use `t.Fatalf` that @LucasRoesler introduced me to, which returns out of the test. 

Also extract `"true"` to `want := "true"` please. 

I'll merge, but send in the alteration after.